app/models/patient.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'chronic'
```

Comment:
Cool library, but I don't see where it was added to the Gemfile?

Reply:
Ah, it is a dependency of `whenever` gem, but I should still add it to the Gemfile.